packages/stark-core/src/modules/http/services/http.service.ts

```diff
@@ -1,16 +1,7 @@
 const _cloneDeep: Function = require("lodash/cloneDeep");
 import { Deserialize, Serialize } from "cerialize";
-import { Observable } from "rxjs/Observable";
-import { ErrorObservable } from "rxjs/observable/ErrorObservable";
-import { timer } from "rxjs/observable/timer";
-import { _throw as observableThrow } from "rxjs/observable/throw";
-// FIXME: importing from single entry "rxjs/operators" together with webpack's scope hoisting prevents dead code removal
-// see https://github.com/ReactiveX/rxjs/issues/2981
-// import { catchError, map, mergeMap, retryWhen } from "rxjs/operators";
-import { catchError } from "rxjs/operators/catchError";
-import { map } from "rxjs/operators/map";
-import { retryWhen } from "rxjs/operators/retryWhen";
-import { mergeMap } from "rxjs/operators/mergeMap";
+import { Observable, timer, throwError } from "rxjs";
+import { catchError, map, mergeMap, retryWhen } from "rxjs/operators";
```

Comment:
Maybe you can add a FIXME for the mergeMap operator to refactor the code where is used so that we fully aling with the RXJS Migration guide.

Reply:
done